spec/mxmnd_spec.rb

```diff
@@ -1,7 +1,64 @@
-require "spec_helper"
+require 'spec_helper'
 
 describe Mxmnd do
   it 'raises BadRequest on missing IP address' do
-    expect { Mxmnd::city(nil) }.to raise_error Mxmnd::BadRequest
+    expect { Mxmnd.city(nil) }.to raise_error Mxmnd::BadRequest
+  end
+end
+
+describe 'Mxmnd.city response' do
+  before {
+  	url = 'https://geoip.maxmind.com/geoip/v2.1/city/45.17.22.11'
+    path = './spec/features/mxmnd_res.txt'
+    ruby_headers = {
```

Comment:
these headers are not ruby-specific, so better just call them `headers` or `request headers`